domains/_vercel.sneazy25.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "sneazy25",
+        "email": "sneazy25@proton.mail"
```

```suggestion
        "email": "sneazy25@protonmail.com"
```

`proton.mail` is not a valid domain name.